cmd/cli/version_latest_test.go

```diff
@@ -17,11 +17,11 @@ func TestOutputLatestReleaseVersion(t *testing.T) {
 		output  string
 	}{
 		{
-			name:    "invalid current version",
-			current: "v1.0.0.0",
+			name:    "customized current version",
```

Comment:
Could you add a new test to the table instead of updating the existing scenario.

Reply:
The existing scenario will fail with the change, as we are going to accept all customized version. I can add a new test and change the current test case accordingly. Please let me know if you have any other questions or suggestions. Thanks